e2e/playwright.config.ts

```diff
@@ -4,6 +4,7 @@ import { TestOptions } from '@approved-premises/e2e'
 
 export default defineConfig<TestOptions>({
   testDir: './',
+  testMatch: /.*\.*(setup)|(spec)\.ts/,
```

Comment:
I think the setup steps need to remain as they are, as they are necessary to ensure the user is first logged out before running any test -- these special steps run _before_ any other feature is started.

Reply:
Let's discuss. I had assumed 'dependencies' just meant 'config to merge in' but if it's affecting the 'pre tests' than yes this needs a rethink

Reply:
No need to discuss, i've put the setup projects back in